docs/cloud-native-security/cspm-get-started-gcp.asciidoc

```diff
@@ -59,8 +59,15 @@ For most users, the simplest option is to use a Google Cloud Shell script to aut
 +
 image::images/cspm-cloudshell-trust.png[The cloud shell confirmation popup]
 +
+NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following pop-up:
++
+image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
++
+If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```

Comment:
Inconsistency between `pop-up` and `popup`, but we could just call it a "message" instead. The fact that it pops up isn't super important.
```suggestion
NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following message:
+
image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
+
If the message appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```